docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+services:
+  ldk-server:
+    container_name: ldk-server
+    build:
+      context: .
+      args:
+        BUILD_FEATURES: ""
+    command: [
+      "--node-listening-address=0.0.0.0:3001",
+      "--node-rest-service-address=0.0.0.0:3000",
+      "--bitcoind-rpc-host=ldk-server-bitcoin",
+      "--bitcoind-rpc-port=18443",
+      "--bitcoind-rpc-user=user",
+      "--bitcoind-rpc-password=pass",
+    ]
+    ports:
+      - "3000:3000"
+      - "3001:3001"
+    networks:
+      - ldk-server
+    depends_on:
+      bitcoin:
+        condition: service_healthy
+    profiles: ["ldk-server"]
+
+  bitcoin:
+    container_name: ldk-server-bitcoin
+    image: blockstream/bitcoind:29.1@sha256:9fcffa83feed0fc382dfb2f26990ca07656d150ba49434096a5aeedac6695a01
```

Comment:
Would be cool to go for v30 directly here, just pinged somebody from Blockstream to check if they intend to upgrade the Docker image soon.

Reply:
And there it is: https://hub.docker.com/r/blockstream/bitcoind

Please update!